### ISSUE TITLE
add-github-page-for-cerebros-enterprise-for-customers

### ISSUE THREAD

**Issue by david-thrower** [david-thrower/cerebros-core-algorithm-alpha#40]
1. New repo
2. Depends on SLA
3. Depends on #39 
4. Depends on #37 
5. Depends on #36 
6. Depends on #34 
7. Add documentation for the enterprise version.

**Comment by david-thrower** [comment]
Completed, needs a V1 revision, included in another issue.
